origin.spec

```diff
@@ -40,7 +39,6 @@
 %global make_redistributable 0
 %endif
 %endif
-}
 
```

Comment:
I think that seems reasonable -- if you patch it into OSE I'll pick up the new commit in this PR